vim/vim.symlink/plugin/vim-tmux-navigator.vim

```diff
@@ -0,0 +1,2 @@
+" Workaround for slow startup time with Fish: https://git.io/vS5cq
+set shell=/bin/bash\ -i
```

Comment:
Is this necessary in conjunction with [this line](https://github.com/bswinnerton/dotfiles/pull/11/files#diff-edbe3b236fe65b305ca6cbc2f5fa585dR3)?